### DISCUSSION TITLE
Database front-end project(s)?

### DISCUSSION THREAD

**Discussion by ChibaPet** [azerothcore/azerothcore-wotlk#11925] [Tools]
Hi all. I'm new to AzerothCore but I've been running it for a few months 
and I'm quite liking it.

I'd like to have a local instance of a web front-end to the database
running, much like https://wotlk.evowow.com/ or https://wotlkdb.com/. Does
such a thing exist tailored for Azerothcore? More generally, is the web
front-end code available somewhere publically, whether for AzerothCore or 
another server? I'm not finding it.

Ideally, I'd like to stand up an existing codebase and then see about 
extending it a bit - seems like it might be useful for folks to be able to
look up individual characters and their tradeskills, for instance.

Thanks in advance for clues!

**Comment by Winfidonarleyan** [comment]
https://github.com/Sarjuuk/aowow Base repo
https://github.com/azerothcore/aowow AC repo

**Reply by ChibaPet** [reply]
Awesome. I somehow passed right over this, repeatedly. Thank you.
